test/boostTest.cpp

```diff
@@ -109,7 +110,9 @@ int registerTests(
 					}
 					catch (boost::exception const& _e)
 					{
-						BOOST_ERROR("Exception during extracted test: " << boost::diagnostic_information(_e));
+						auto const* semanticTest = dynamic_cast<SemanticTest const*>(testCase.get());
```

Comment:
I think we should instead have a try/catch inside the semantic test.

Reply:
I agree, was thinking to do like that